tests/test_dsp.py

```diff
@@ -93,6 +93,34 @@ def test_group_delay_single_channel(impulse_group_delay):
         grp, impulse_group_delay[1].flatten(), rtol=1e-10, atol=1e-10)
 
 
+def test_complex_group_delay_single_channel(impulse_complex_group_delay):
+    """Test the function returning the group delay of a signal,
+    single channel."""
+    signal = impulse_complex_group_delay[0]
+
+    with pytest.raises(ValueError, match="Invalid method"):
```

Comment:
I guess the ValueErrors are already tested elsewhere and could be removed here

Reply:
Probably you're right. I just tested the same as for non complex valued signals to be sure we have the same behavior. But I kill these lines...